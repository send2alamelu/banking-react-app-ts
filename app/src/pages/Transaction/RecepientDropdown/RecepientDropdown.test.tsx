import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import RecepientDropDown from './RecepientDropDown';

describe('RecepientDropDown Component', () => {
  afterEach(() => {
    cleanup();
  });

  test('renders RecepientDropDown with children', () => {
    const { asFragment } = render(<RecepientDropDown onChange={() => { }} />);
    expect(asFragment()).toMatchSnapshot();
  });
});