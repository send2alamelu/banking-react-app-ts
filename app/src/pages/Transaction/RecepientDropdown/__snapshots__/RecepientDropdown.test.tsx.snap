// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecepientDropDown Component renders RecepientDropDown with children 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-1l9hhr"
  >
    <div
      class="MuiFormControl-root MuiFormControl-fullWidth css-q8hpuo-MuiFormControl-root"
    >
      <label
        class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
        data-shrink="false"
        id="demo-simple-select-label"
      >
        Recepient
      </label>
      <div
        class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl MuiSelect-root css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
      >
        <div
          aria-expanded="false"
          aria-haspopup="listbox"
          aria-labelledby="demo-simple-select-label demo-simple-select"
          class="MuiSelect-select MuiSelect-outlined MuiOutlinedInput-input MuiInputBase-input css-11u53oe-MuiSelect-select-MuiInputBase-input-MuiOutlinedInput-input"
          id="demo-simple-select"
          role="button"
          tabindex="0"
        >
          <span
            class="notranslate"
          >
            â€‹
          </span>
        </div>
        <input
          aria-hidden="true"
          class="MuiSelect-nativeInput css-yf8vq0-MuiSelect-nativeInput"
          tabindex="-1"
          value=""
        />
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiSelect-icon MuiSelect-iconOutlined css-hfutr2-MuiSvgIcon-root-MuiSelect-icon"
          data-testid="ArrowDropDownIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M7 10l5 5 5-5z"
          />
        </svg>
        <fieldset
          aria-hidden="true"
          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
        >
          <legend
            class="css-dhh0px"
          >
            <span>
              Recepient
            </span>
          </legend>
        </fieldset>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
